### PR TITLE
[awsecscontainermetrics] receiver- Update README

### PR DESCRIPTION
**Pull request by hossain-rayhan**
Update README to include full configuration examples.

### CODE REVIEW

File: receiver/awsecscontainermetricsreceiver/README.md [hunk 1]
```diff
@@ -3,21 +3,173 @@
...
+------------ | ------------- | --------------------
+ecs.task.memory.usage | container.memory.usage | Bytes
+ecs.task.memory.usage.max | container.memory.usage.max | Bytes
+ecs.task.memory.usage.limit | container.memory.usage.limit | Bytes
+ecs.task.memory.reserved | Bcontainer.memory.reserved | MB
+ecs.task.memory.utilized | container.memory.utilized | MB
+ecs.task.cpu.usage.total | container.cpu.usage.total | NS
+ecs.task.cpu.usage.kernelmode | container.cpu.usage.kernelmode | NS
+ecs.task.cpu.usage.usermode | container.cpu.usage.usermode | NS
+ecs.task.cpu.usage.system | container.cpu.usage.system | NS
```
Comment (PettitWesley) [inline receiver/awsecscontainermetricsreceiver/README.md:41]: NS means nanoseconds?? 

I've never seen "NS" before. I have seen "ns" lower case though

File: receiver/awsecscontainermetricsreceiver/README.md [hunk 2]
```diff
@@ -3,21 +3,173 @@
...
+------------ | -------------
+ecs.cluster | ecs.cluster
+ecs.task-definition-family | ecs.task-definition-family
+ecs.task-arn | ecs.task-arn
+ecs.task-id | ecs.task-id
+ecs.task-definition-version | ecs.task-definition-version
+ecs.service | ecs.service
+"" | `container.name`
+"" | `container.id`
+"" | `ecs.docker-name`
```
Comment (PettitWesley) [inline receiver/awsecscontainermetricsreceiver/README.md:74]: These rows look weird when I view them...


Its also odd to have this as a table, a list might make more sense.

Either way, I think the container attributes shouldn't have special formatting, and the empty strings look weird.

File: receiver/awsecscontainermetricsreceiver/README.md (hunk 1 above)
Comment (hossain-rayhan) [reply receiver/awsecscontainermetricsreceiver/README.md]: Used full form of units matching with CloudWatch unit conventions. https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_MetricDatum.html

File: receiver/awsecscontainermetricsreceiver/README.md (hunk 2 above)
Comment (hossain-rayhan) [reply receiver/awsecscontainermetricsreceiver/README.md]: Updated.

**Review by PettitWesley — APPROVED** [review]

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**